internal/cli/server_upgrade.go

```diff
@@ -586,5 +620,14 @@ Please run the following commands if you wish to unset these runner profiles
 from being the default:
 
 %[2]s
+`)
+
+	runnerProfileUpgradeConfigError = strings.TrimSpace(`
+The plugin config for runner profile %[1]s failed to correctly be parsed. 
+Plugin Config Error: %[3]s
+
+Please run the following with the corrected plugin configuration to fix this.
+
+waypoint runner profile set -name=%[2]s -plugin-config=<path_to_config_file>
```

Comment:
Should our error message tell people that the format changed starting with `0.9`, which is why it's not parsing correctly?

Reply:
Ah yeah, we should be more specific. The issue was our old server install CLI laid down a invalid on-demand runner profile config for kubernetes, and prior to this release, we never validated Task plugin configs prior to a task being invoked and threw away the config validation error, so if you upgrade with a bad config all future tasks will fail to be launched after upgrades.